numpy/core/fromnumeric.py

```diff
@@ -1063,6 +1064,16 @@ def argsort(a, axis=-1, kind=None, order=None):
     >>> x = np.array([3, 1, 2])
     >>> np.argsort(x)
     array([1, 2, 0])
+    
+    Index of the 4th largest element in an unsorted 1D array.
+    
+    >>> x = np.array(1, 2, 0, 3, 4, 5])
+    >>> np.argsort(x)[-4]    
```

Comment:
The point of these examples is to show the output. I would just show the result of `x[np.argsort(x)]` instead of these two examples.